lib/utils/github/handle/app-updates.ts

```diff
@@ -131,9 +131,10 @@ export default async function handleAppUpdates({
 	// TODO: Weviate should be per project, not per user
 	const vectorDB = new Weaviate(user.id)
 	for (const repo of addedRepos) {
-		const repoUrl = `${GITHUB.BASE_URL}/${repo.full_name}`
-		const branch = await getMainBranch(repo.full_name)
-		await vectorDB.embedRepo(repoUrl, branch)
+		const installationId = await getInstallationId(repo.full_name)
+		const installationToken = await getInstallationToken(installationId)
+		const branch = await getMainBranch(repo.full_name, installationToken)
+		await vectorDB.embedRepo(repo.full_name, branch)
```

Comment:
Since these 4 lines are repeated ~4 times, we could add a helper like `await vectorDB.embedFromInstallation(repo.full_name, branch)` which calls `getInstallationId`, `getInstallationToken`, `getMainBranch`, and `embedRepo` internally